api/src/main/java/io/druid/data/input/impl/TimestampSpec.java

```diff
@@ -92,23 +146,15 @@ public boolean equals(Object o)
     }
 
     TimestampSpec that = (TimestampSpec) o;
-
-    if (!timestampColumn.equals(that.timestampColumn)) {
-      return false;
-    }
-    if (!timestampFormat.equals(that.timestampFormat)) {
-      return false;
-    }
-    return !(missingValue != null ? !missingValue.equals(that.missingValue) : that.missingValue != null);
-
+    return Objects.equal(this.timestampColumn, that.timestampColumn) &&
```

Comment:
any reason we are changing the way we generate hashcodes than the standard intellij auto-generated one?